news/bugfix-3574.md

```diff
@@ -0,0 +1 @@
+`affile`: Fix improper initialization in affile and LogWriter
```

Comment:
I would mention memory leaks here along with the severity of the leaks, so that a user can understand how much her/his installation might be impacted.

Reply:
OK, news file updated.